site/app/templates/polls/PollForm.twig

```diff
@@ -140,20 +140,25 @@
             <option value="when_ended" {{ poll is not null and poll.getReleaseAnswer() == 'when_ended' ? "selected" : "" }}>After Poll Ended</option>
             <option value="always" {{ poll is not null and poll.getReleaseAnswer() == 'always' ? "selected" : "" }}>Always</option>
         </select>
-        <br/>
-
+        <br><br>
+        <label for = "poll_custom_options" class="option-title credit-form-item">
+            Allow custom answers from students
+        </label>
+        <input id ="poll_custom_options" class="credit-box credit-form-item" type="checkbox" name="poll_custom_options"  {{ poll is not null and poll.getAllowsCustomResponses() ? "checked" : "" }}>
```

Comment:
The ID and name should both be dash-separated as per the [style guide](https://submitty.org/developer/software_and_system_design/coding_style_guide/css#styling).  Lots of small whitespace issues here too.